mpconstants/carriers.py

```diff
@@ -119,4 +119,312 @@
     }
 }
 
```

Comment:
The variable would be `CARRIER_DETAILS['ORANGE'].slug` instead of `'orange'`, so I don't know if that's really an improvement... 

The alternate route would be to store this in a property for each carrier in `CARRIER_DETAILS`, but I ended up not doing that because the function to generate the list in the format we need was super ugly and we never needed to access it in a different way.